### ISSUE TITLE
`Zpl Printer is not admited`, Is this the end?

### ISSUE THREAD

**Issue by parallels999** [sbinkert/ZplPrinter#7]
![image](https://user-images.githubusercontent.com/109294935/200870661-59dfc1e2-f93e-4bd0-854c-dbd5c4f89b2c.png)

Migrating Chrome Apps to Progressive Web Apps
--
Hi **Everyone**, 

We'd like to remind you of the [previously communicated](https://blog.chromium.org/2021/10/extending-chrome-app-support-on-chrome.html) timeline for the end of support for Chrome Apps. Based on customer and partner feedback, we are re-confirming the expansion of our phased approach for ending the support of Chrome Apps. 

Below we've included the timeline for the phased approach for ending the support of Chrome Apps:

- June 2022: Chrome Web Store will stop offering Chrome App installations for Windows, Mac, and Linux users.
- July 2022: End support for Chrome Apps on Windows, Mac, and Linux.
- March 2023: Chrome Apps will no longer be launchable on Windows, Mac, and Linux.
- January 2025: End support for Chrome Apps on ChromeOS.

**Source:** [chrome_app_deprecation](https://support.google.com/chrome/thread/174381169?visit_id=638036034058848176-339152761&p=chrome_app_deprecation&rd=1)

**Comment by MrL0co** [comment]
For anyone still looking for an alternative i have forked this project and wrapped it in a Electron app so it can now run as a standalone application. I like to ask @sbinkert if he agrees to allow me to now host what is still essentially his code on my own repository, it has a fully automated deployment setup and a major parent company likes to use my binaries to distribute to there employees.

https://github.com/MrL0co/ZplPrinter

**Comment by sbinkert** [comment]
> For anyone still looking for an alternative i have forked this project and wrapped it in a Electron app so it can now run as a standalone application. I like to ask @sbinkert if he agrees to allow me to now host what is still essentially his code on my own repository, it has a fully automated deployment setup and a major parent company likes to use my binaries to distribute to there employees.
> 
> https://github.com/MrL0co/ZplPrinter

I agree and allow any kind of re-use of my entire source code in the Zpl Printer repo.  
@MrL0co Thanks for your effort and the port to electron app!
